subsys/net/lib/nrf_cloud/src/nrf_cloud_fsm.c

```diff
@@ -16,6 +16,10 @@
 #include <net/nrf_cloud_pgps.h>
 #endif
 
+#ifndef EFTYPE
```

Comment:
can you instead change the return value of `static int location_process(const char * const buf)` from EFTYPE to ENOSYS?

Reply:
done